admin.php

```diff
@@ -290,6 +290,9 @@ public function onPluginsInitialized()
             };
         }
 
+        // Load utility class
+        require_once __DIR__ . '/classes/utils.php';
```

Comment:
Added my change in https://github.com/getgrav/grav-plugin-admin/pull/686/commits/b33fd299fe31648878e4b6c5d0b683ee7ed7d892, what you think?